.github/workflows/release.yml

```diff
@@ -100,7 +100,8 @@ jobs:
   release:
     strategy:
       matrix:
-        version: [15]
+        arch: [amd64, arm64]
+        version: [14, 15, 16]
```

Comment:
Might be worth looking at [this](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations).

I think this would also work:
```yml
      matrix:
        arch: [amd64, arm64]
        version: [14, 15, 16]
        include:
          - arch: amd64
            full_arch: x86_64
          - arch: arm64
            full_arch: aarch64
```
Then you could delete the step to set the env and just use `${{ matrix.full_arch }}`

Reply:
Thanks